packages/0xcert-ethereum-erc721-contracts/src/contracts/nf-token-metadata.sol

```diff
@@ -45,9 +45,14 @@ contract NFTokenMetadata is
   string internal nftSymbol;
 
   /**
-   * @dev URI base for NFT metadata. NFT URI is made from base + NFT id.
+   * @dev URI prefix for NFT metadata. NFT URI is made from prefix + NFT id + postfix.
    */
-  string public uriBase;
+  string public uriPrefix;
+
+  /**
+   * @dev URI postfix for NFT metadata. NFT URI is made from prefix + NFT id + postfix.
```

Comment:
* @dev URI postfix for NFT metadata. NFT URI is made from prefix + NFT ID + postfix.